src/mcp/prompts/crashlytics/connect.ts

```diff
@@ -105,12 +124,15 @@ Follow these steps to diagnose and fix issues.
     <A plan for how to test that the issue has been fixed and protect against regressions>
       1. <Test case 1>
       2. <Test case 2>
+
+    ## Other potential causes
+    1. <Another possible root cause>
+    2. <Another possible root cause>
       
-  7. Present the plan to the user and get approval before making the change.
-  8. Fix the issue.
-    8a. Be mindful of API contracts and do not add fields to resources without a clear way to populate those fields
-    8b. If there is not enough information in the crash report to find a root cause, describe why you cannot fix the issue instead of making a guess.
-  9. Ask the developer if they would like you to test the fix for them.
+  9. Present the plan to the user and get approval before making the change.
+  10. Only if they approve the plan, create a fix for the issue.
```

Comment:
Let's add a statement to honor the license of the source code if any! Eg: suggesting something from an external library/dependency without proper license could put the developer under risks.

Reply:
Agreed! But, when it happens - it could be bad. We don't have to do this now, but something for us to bear in mind for a future change.